plugins/main/public/controllers/management/components/management/statistics/hooks/use-build-statistics-visualizations.ts

```diff
@@ -10,20 +10,25 @@
  *
  * Find more information about this on the LICENSE file.
  */
-import { useEffect } from "react";
+import { useEffect } from 'react';
 
-import store from "../../../../../../redux/store";
-import { TabVisualizations } from "../../../../../../factories/tab-visualizations";
-import { LoadedVisualizations } from "../../../../../../factories/loaded-visualizations";
-import { DiscoverPendingUpdates } from "../../../../../../factories/discover-pending-updates";
-import { RawVisualizations } from "../../../../../../factories/raw-visualizations";
-import { GenericRequest } from "../../../../../../react-services/generic-request";
-import { updateVis } from "../../../../../../redux/actions/visualizationsActions";
-import { AppState } from "../../../../../../react-services/app-state";
+import store from '../../../../../../redux/store';
+import { TabVisualizations } from '../../../../../../factories/tab-visualizations';
+import { LoadedVisualizations } from '../../../../../../factories/loaded-visualizations';
+import { DiscoverPendingUpdates } from '../../../../../../factories/discover-pending-updates';
+import { RawVisualizations } from '../../../../../../factories/raw-visualizations';
+import { GenericRequest } from '../../../../../../react-services/generic-request';
+import { updateVis } from '../../../../../../redux/actions/visualizationsActions';
+import { AppState } from '../../../../../../react-services/app-state';
 import { WazuhConfig } from '../../../../../../react-services/wazuh-config.js';
+import { StatisticsDataSource } from '../../../../../../components/common/data-source/pattern/statistics/index.js';
 
-export const useBuildStatisticsVisualizations = (clusterNodeSelected, refreshVisualizations) => {
-  const { 'cron.prefix': indexPrefix, 'cron.statistics.index.name': indexName } = new WazuhConfig().getConfig();
+export const useBuildStatisticsVisualizations = (
```

Comment:
**suggestion**: this hook seems that is not used anymore. We could remove it.

Reply:
Deleted unused  `use-build-statistics-visualizations` hook.